routes/web.php

```diff
@@ -11,24 +11,35 @@
 */
 
 
-Route::get('/', function () {
-    return view('welcome');
-})->middleware('guest');
+// Route::get('/', function () {
+//     return view('welcome');
+// })->middleware('guest');
+
+
+Route::get('/','HomeController@show' );
 
 // remove later
 Route::get('/upload', function () {
     return view('modules.upload');
 });
 
-Auth::routes();
 
-Route::get('/dashboard', 'DashboardController@index')->name('dashboard');
+Auth::routes(); // includes routes for login, register, forget password
+
+// only authenticated user can access this route group
+Route::group(['prefix'=>'dashboard', 'middleware' => 'auth'], function () {
```

Comment:
can define both route permission and authorization permission, e.g (user ability and model policy)